.cirrus.yml

```diff
@@ -19,6 +19,38 @@ task:
   build_script: |
     make -C scripts/ci vagrant-fedora-no-vdso
 
+task:
+  name: CentOS Stream 9 based test
+  environment:
+    HOME: "/root"
+    CIRRUS_WORKING_DIR: "/tmp/criu"
+
+  compute_engine_instance:
+    image_project: centos-cloud
+    image: family/centos-stream-9
+    platform: linux
+    cpu: 4
+    memory: 8G
+
+  setup_script: |
+    ln -sf /usr/include/google/protobuf/descriptor.proto images/google/protobuf/descriptor.proto
+    dnf config-manager --set-enabled crb # Same as CentOS 8 powertools
+    dnf -y install epel-release epel-next-release
+    dnf -y install --allowerasing asciidoc gcc git gnutls-devel libaio-devel libasan libcap-devel libnet-devel libnl3-devel libbsd-devel libselinux-devel make protobuf-c-devel protobuf-devel python3-devel python3-PyYAML python3-future python3-protobuf python3-junit_xml xmlto
+    pip install flake8
+    systemctl stop sssd
+    # Even with selinux in permissive mode the selinux tests will be executed
+    # The Cirrus CI user runs as a service from selinux point of view and is
+    # much more restricted than a normal shell (system_u:system_r:unconfined_service_t:s0)
+    # The test case above (vagrant-fedora-no-vdso) should run selinux tests in enforcing mode
+    setenforce 0
+    # TODO: remove pip3 install, add python3-flake8 to dnf install above
+    # once https://bugzilla.redhat.com/show_bug.cgi?id=2070206 is fixed.
+    pip3 install flake8
```

Comment:
And here pip is installed a second time. Probably the first can be dropped.

Reply:
fixed, thanks!